modules/manage/pages/node-management.adoc

```diff
@@ -22,7 +22,7 @@ When a broker is in maintenance mode, Redpanda continues to replicate updates to
 
 == Place a broker in maintenance mode
 
-Before placing a broker in maintenance mode, you may want to temporarily disable or ignore alerts related to under-replicated partitions. To prevent under-replicated partitions altogether, you can move all partitions to other brokers.
+Before placing a broker in maintenance mode, you may want to temporarily disable or ignore alerts related to under-replicated partitions. These alerts likely point to false positives as a result of the broker being taken offline and replicas being temporarily unreachable. To prevent under-replicated partitions altogether, you can move all partitions to other brokers.
```

Comment:
Maybe we can phrase it like this: 
`Before shutting down the broker, you may want...` ?